applications/asset_tracker_v2/boards/nrf9160dk_nrf9160ns.conf

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nrf9169dk.
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# CAF - Common Application Framework
+CONFIG_GPIO=y
+CONFIG_PWM=n
+CONFIG_LED_GPIO=y
+CONFIG_LED_PWM=n
+CONFIG_CAF_LEDS_GPIO=y
```

Comment:
CONFIG_CAF_LEDS_PWM is part of choice (neither of its options is enabled by default) so we don't need to explicitly disable it.
CONFIG_CAF_LEDS_GPIO=y is enough

We also don't need to explicitly disable CONFIG_LED_PWM or CONFIG_PWM. Unless it was enabled by default in board config and we want to shrink footprint. Please check if this is needed and remove the lines if not.